lam4doc/entrypoints/api/handlers.py

```diff
@@ -69,6 +70,29 @@ def generate_indexes() -> tuple:
         raise InternalServerError(str(e))  # 500
 
 
+def get_lam_files() -> tuple:
+    """
+    API method for generating and requesting all LAM files.
+    :rtype: report file (zip), int
+    :return: all lam files: HTML and PDF report, and 3 index files
+    """
+    logger.debug('start get lam files endpoint')
+    try:
+        with tempfile.TemporaryDirectory() as temp_folder:
+            files_to_zip = list()
+            files_to_zip.append(service_generate_lam_report(temp_folder, HTML_REPORT_TYPE))
+            files_to_zip.append(service_generate_lam_report(temp_folder, PDF_REPORT_TYPE))
+            files_to_zip += service_generate_indexes(temp_folder)
+
+            archive = zip_files(temp_folder, files_to_zip, 'indexes.zip')
```

Comment:
filename can be a constant somewhere to easier change it; I would call the archive `lam_assets.zip`